packages/editor/src/client/blocks/types.ts

```diff
@@ -467,13 +469,15 @@ export function unionMapForBlock(block: BlockValue): BlockInput {
 
               case TeaserType.Page:
                 return {
-                  page: {
-                    style: flexTeaser.teaser.style,
-                    imageID: flexTeaser.teaser.image?.id,
-                    preTitle: flexTeaser.teaser.preTitle || undefined,
-                    title: flexTeaser.teaser.title || undefined,
-                    lead: flexTeaser.teaser.lead || undefined,
-                    pageID: flexTeaser.teaser.page.id
+                  teaser: {
+                    page: {
```

Comment:
what about the `teaser: { article: ...}}` did you already implement that?

Reply:
Yes that was already in place.